filter-10-secure-ssh.conf

```
@@ -21,5 +20,11 @@ filter {
       tag_on_failure => ["_grokparsefaillure","pam_unix_session_grok_failed"]
     }
   }
+   if [message] =~ /^Did not receive identification string/ {
+    grok {
+      match => ["message","Did not receive identification string from %{IPORHOST:[client][ip]} port %{NUMBER:[client][port]}"]
+      id => "ssh_monitoring"
```

```suggestion
      id => "ssh_monitoring_noidentstring"
```
I have to admit, my new name is very ugly. I just wanted to show that the `id` has to be unique for the whole pipeline, so `ssh_monitoring` could be a bit too generic. Since `id` is only visible in API and Monitoring, it's ok to go with this name now and change it later, though.